client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/EditInteractionEventDialog/EditInteractionEventDialog.tsx

```diff
@@ -38,7 +39,11 @@ const EditInteractionEventDialog : React.FC<Props> = (props: Props) : JSX.Elemen
     }),
         [interactionEventDialogData, setInteractionEventDialogData]);
 
-    const onConfirm = () => editInteractionEvent(interactionEventDialogData)
+    const onConfirm = () => editInteractionEvent(
+        {...interactionEventDialogData, 
+            placeName: interactionEventDialogData.placeName ? interactionEventDialogData.placeName : defaultPlaceName
```

Comment:
doesnt really matter but could be  interactionEventDialogData.placeName || defaultPlaceName.

Reply:
also default place name can be generated here according to interactionEventDialogData.place type and not on every place type change - that might also save the passing-down of the setdefault func

Reply:
I will change it to be interactionEventDialogData.placeName || defaultPlaceName but about the second comment the problem with generating it here is that I don't have the subType's dispaly name here but only its code